data/json/items/tools.json

```diff
@@ -90,7 +90,33 @@
     "color": "light_gray",
     "ammo": "battery",
     "max_charges": 1000,
-    "flags": [ "NO_UNLOAD", "RECHARGE" ]
+    "covers": [ "TORSO" ],
+    "encumbrance": 9,
+    "coverage": 15,
+    "material_thickness": 4,
+    "flags": [ "NO_UNLOAD", "RECHARGE", "BELTED", "UPS", "OVERSIZE" ]
+  },
+  {
+    "id": "UPS_Engineer_off",
+    "type": "TOOL_ARMOR",
+    "name": "engineer's UPS",
+    "name_plural": "engineers' UPS's",
+    "description": "This is a unified power supply, or UPS. The Engineer's UPS is designed to power the hungriest of UPS devices.",
+    "weight": 62000,
```

Comment:
Wearable 62-kg tool? That's not Engineer's UPS, but Superman's UPS.

Reply:
I think it might just be a balance thing, but I'm not too good at that.
I'll have to double check the other UPS's.

Reply:
Oh Lordie.
It was probably only supposed to be 6800, not 62000....